modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp

```diff
@@ -134,12 +205,35 @@ portletDisplay.setURLBack(backURL);
 						});
 
 						if (shouldSubmitForm) {
-							const values = <portlet:namespace />getValues(fields);
+							let values = <portlet:namespace />getValues(fields);
+							const categoriesContent = document.getElementById(
+								'<portlet:namespace />categorization'
+							);
 							const externalReferenceCode = <portlet:namespace />getExternalReferenceCode();
 							const path = <portlet:namespace />getPath(
 								externalReferenceCode
 							);
 
+							if (categoriesContent) {
+								const categoryIds = Array.from(
```

Comment:
Hey @seltong, please create function to reuse the code, like this one: 

```
function <portlet:namespace />getInputValues(selector) {
	return Array.from(
		categoriesContent.querySelectorAll(selector)
	).map((item) => item.value);
}
```

and then apply it.

```
const categoryIds = <portlet:namespace />getInputValues('input[name^="<portlet:namespace />assetCategoryIds"]');

const tagNames = <portlet:namespace />getInputValues('input[name^="<portlet:namespace />assetTagNames"]');

values = Object.assign(
	values,
	{['categoryIds']: categoryIds},
	{['tagNames']: tagNames}
);
```
I don't know if we can inline the calls to `getInputValues` in the `Object.assign` call, can you please check this?

Thx

Reply:
Yeah, of course. I will create this function and check this

Thx